stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/feature/messages/composer/MessageComposerController.kt

```diff
@@ -813,90 +812,18 @@ public class MessageComposerController(
      * Shows the mention suggestion list popup if necessary.
      */
     private suspend fun handleMentionSuggestions() {
+        // TODO double-check if we need to use the regex pattern to check for mentions
         val containsMention = MentionPattern.matcher(messageText).find()
-
+        // val containsMention = messageText.startsWith(MentionStartChar)
```

Comment:
It will only work with mentions that are typed with an empty message.
If you start to type anything and then after that introduce a `@`, it won't match.
eg: `Hi @kanat`

Reply:
+ transliterator won't work with the current pattern